### ISSUE TITLE
Proposal - A verifier should have an 'employee type' and an address

### ISSUE THREAD

**Issue by gratiartis** [kittylyst/VillagePaymentAgent#69]
It seems that in the current domain model, an `Address` has a `Verifier`, but a `Verifier` does not have an `Address`. To support a query of addresses of employees in a particular region, this seems to have led to a situation where the 'employee type' is a property of an address (presumably to avoid joining the two tables).

In terms of a domain model, it seems to me that it would make more sense for a `Verifier` to have an `employeeType` and an `Address` (which may also have a reciprocal `Verifier`). So if I wish to query for employed verifiers in a particular region, I would use a finder in the `VerifierDao`.

**Comment by njacobs5074** [comment]
Yes, this makes sense.  Will need some further review, but I get what you're saying.
